package.json

```diff
@@ -19,17 +15,13 @@
     "release": "standard-version -s",
     "postrelease": "npm publish && git push --follow-tags",
     "pretest": "standard",
-    "test": "tap -J --100 test/*.js",
-    "update-coc": "weallbehave -o . && git add CODE_OF_CONDUCT.md && git commit -m 'docs(coc): updated CODE_OF_CONDUCT.md'",
-    "update-contrib": "weallcontribute -o . && git add CONTRIBUTING.md && git commit -m 'docs(contributing): updated CONTRIBUTING.md'"
+    "test": "tap -J --100 test/*.js"
```

Comment:
I think you can have this just be `"test": "tap"`, and then add a section like:

```json
{
  "tap": {
    "check-coverage": true
  }
}
```

And rename `test/util` to `test/fixtures` so that tap will ignore it.